### PR TITLE
CORE-2006: updated dependencies and fixed some lint warnings

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -20,16 +20,14 @@
...
+                 [cheshire "5.13.0"]
+                 [medley "1.4.0"]
                  [slingshot "0.12.2"]
-                 [org.cyverse/otel "0.2.5"]
-                 [org.cyverse/clojure-commons "3.0.6"]]
+                 [org.cyverse/otel "0.2.6"]
+                 [org.cyverse/clojure-commons "3.0.8-SNAPSHOT"]]
   :profiles {:repl {:source-paths ["repl"]}}
-  :eastwood {:exclude-linters [:unlimited-use]}
+  :eastwood {:exclude-linters [:unlimited-use :non-dynamic-earmuffs]}
```
Comment (ianmcorvidae) [inline project.clj:29]: :no_entry_sign: :headphones: 

:laughing:

File: src/clj_jargon/item_info.clj [hunk 2]
```diff
@@ -109,14 +109,11 @@
...
-  [{^IRODSFileFactory file-factory :fileFactory} ^String path]
+  [{^IRODSFileSystemAO cm-ao :fileSystemAO} ^String path]
   (otel/with-span [s ["is-linked-dir?" {:attributes {"irods.path" path}}]]
     (validate-path-lengths path)
     (= ObjStat$SpecColType/LINKED_COLL
-       (.. file-factory
-         (instanceIRODSFile (ft/rm-last-slash path))
-         initializeObjStatForFile
-         getSpecColType))))
+       (.getSpecColType (.getObjStat cm-ao path)))))
```
Comment (ianmcorvidae) [inline src/clj_jargon/item_info.clj:116]: :+1:

**Review by ianmcorvidae — APPROVED** [review]
All looks good to me!

**Comment by slr71** [comment]
Thanks for the review! 👍

**Pull request merged**